src/pymovements/utils/archives.py

```diff
@@ -252,14 +252,15 @@ def _decompress(
     ----------
     source_path : Path
         Path to the file to be decompressed.
-    destination_path : Path
+    destination_path : Path | None
         Path to the decompressed file.
     remove_finished : bool
         If ``True``, remove the source file after decompression.
 
     Returns
     -------
-    Path : Path to the decompressed file.
+    Path : Path
```

Comment:
is there a reason for naming this return value Path?

is pydoclint complaining about the line otherwise?

maybe without the `:`?